demos/RemoteAttestation/README.md

```diff
@@ -0,0 +1,28 @@
+This project demonstrates how to run FAKE RemoteAttestation on hyperEnclave, which equipped with FAKE TPM and no CA support. The RemoteAttestation equipped with hardware TPM and CA are not open source now.
```

Comment:
How about:

which is equipped with FAKE TPM and is without the CA support?

Reply:
Thanks, done.